docs/user-guide/continuous-aggregation/manage-flow.md

```diff
@@ -8,6 +8,28 @@ Each `flow` is a continuous aggregation query in GreptimeDB.
 It continuously updates the aggregated data based on the incoming data.
 This document describes how to create, and delete a flow.
 
+## Create input table
+
+Before creating a flow, you need to create a source table to store the raw data. Like this:
+```sql
+CREATE TABLE temp_sensor_data (
+  sensor_id INT,
+  loc STRING,
+  temperature DOUBLE,
+  ts TIMESTAMP
+);
+```
+However, if you don't want to store the raw data, you can use a temporary table as the source table by creating table using `WITH ('ttl' = 'instant')` table option:
+```sql
+CREATE TABLE temp_sensor_data (
+  sensor_id INT,
+  loc STRING,
+  temperature DOUBLE,
+  ts TIMESTAMP
+) WITH ('ttl' = 'instant');
+```
+setting `'ttl'` to `'instant'` will make the table a temporary table, which means it will automatically discard all inserted data and the table will always be empty, only sending them to flow task for computation.
```

Comment:
```suggestion
Setting `'ttl'` to `'instant'` will make the table a temporary table, which means it will automatically discard all inserted data and the table will always be empty, only sending them to flow task for computation.
```